src/sage/schemes/elliptic_curves/period_lattice.py

```diff
@@ -977,16 +977,16 @@ def basis_matrix(self, prec=None, normalised=False):
 
         INPUT:
 
-        - ``prec`` (int or ``None``(default)) -- real precision in
+        - ``prec`` (int or ``None`` (default)) -- real precision in
           bits (default real precision if ``None``).
 
-        - ``normalised`` (bool, default False) -- if True and the
+        - ``normalised`` (bool, default ``False``) -- if ``True`` and the
           embedding is real, use the normalised basis (see
           ``normalised_basis()``) instead of the default.
```

Comment:
should we do sphinx syntax here for `normalised_basis()`?

Reply:
yes

Reply:
done in dab424b4a68